### DISCUSSION TITLE
Identify records uniquely

### DISCUSSION THREAD

**Discussion by moritzvolkmar** [Samedis-care/samedis-care-community#60] [Shipped - Live on production]
Data records should be uniquely identifiable over time (multi-year maintenance intervals).
Inventory numbers tend to change at larger facilities. If you export the inventory list, you don't have a unique reference at the moment.

**Comment by YvesR** [comment]
I do not follow the description, please add more infos and example.

**Reply by moritzvolkmar** [reply]
Actually, it is only a matter of optionally outputting the Samedis.care ID of the data records.

**Reply by moritzvolkmar** [reply]
When customer uses Samedis.care to monitor his devices and over the years hires different service providers to inventory the devices and they do it according to their own nomenclatures. Then e.g. the original device 1 was device 1895 in between and is now device 13.

**Reply by moritzvolkmar** [reply]
However, this only makes sense in connection with third-party systems.
